great_expectations/rule_based_profiler/parameter_builder/parameter_builder.py

```diff
@@ -181,6 +177,26 @@ def get_metrics(
         variables: Optional[ParameterContainer] = None,
         parameters: Optional[Dict[str, ParameterContainer]] = None,
     ) -> MetricComputationResult:
+        """
+        General multi-batch metric computation facility.
+
+        Computes specified metric (can be multi-dimensional, numeric, non-numeric, or mixed) and conditions (or
+        "sanitizes") result according to two criteria: enforcing metric output to be numeric and handling NaN values.
+        :param batch_ids: IDs of Batch objects used to compute the metric -- commonly obtained via the "get_batch_ids()"
+        method in this module, although it can readily accept the list of Batch IDs generated through any other means.
+        :param validator: the Validator used for metric calculation purposes.
+        :param metric_name: Name of metric of interest, being computed.
+        :param metric_domain_kwargs: Metric Domain Kwargs is an essential parameter of the MetricConfiguration object.
+        :param metric_value_kwargs: Metric Value Kwargs is an essential parameter of the MetricConfiguration object.
+        :param enforce_numeric_metric: Flag controlling whether or not metric output must be numerically-valued.
+        :param replace_nan_with_zero: Directive controlling how NaN metric values, if encountered, should be handled.
+        :param domain: Domain object scoping "$variable"/"$parameter"-style references in configuration and runtime.
+        :param variables: Part of the "rule state" available for "$variable"-style references.
+        :param parameters: Part of the "rule state" available for "$parameter"-style references.
+        :return: MetricComputationResult object, containing both: data samples in the format "N x R^m", where "N" (most
+        significant dimension) is the number of measurements (e.g., one per Batch of data), while "R^m" is the
+        multi-dimensional metric, whose values are being estimated, and details (to be used for metadata purposes).
+        """
```

Comment:
Nit - love the docstring but can we utilize the Google styled ones moving forward?

Don't need to change this since it's not blocking but I think the format from Google is what we've noted in our style guide.

Reply:
How about later on this one? :-)